pkg/render/kubecontrollers/kube-controllers.go

```diff
@@ -487,7 +487,7 @@ func (c *kubeControllersComponent) controllersDeployment() *appsv1.Deployment {
 			},
 		},
 		Spec: appsv1.DeploymentSpec{
-			Replicas: &kubeControllerReplicas,
+			Replicas: &replicas,
```

Comment:
Shouldn't we be adding podAntiAffinity here too?

Reply:
Replicas count is still 1 for kube-controllers so it isn't needed for now.